css-text/Overview.bs

```diff
@@ -1324,6 +1327,11 @@ Canonical order: N/A
       Shaping characters are still shaped as if the word were not
       broken, and grapheme clusters must stay together as one unit.
       No hyphenation character is inserted at the break point.
+    <dt><dfn>break-spaces</dfn></dt>
+    <dd>Line breaks are handled identically to ''overflow-wrap/normal'',
+      except that any sequence of <a>preserved</a> white space
+      that would otherwise overflow the line under these rules
+      must be broken after the last white space character that would fit the line.
```

Comment:
Would this be clearer? 

> Line breaks are handled identically to ''overflow-wrap/normal'',
> except that if a sequence of <a>preserved</a> white space
> would otherwise overflow the line under these rules,
> the line must be broken after the last white space character of that sequence
> that would fit the line.

This is meant to be the same behavior, but should be more explicit. Break in places unrelated to the white space sequence are allowed as usual; it allows the break to happen after the last character in the sequence. It does not create a break opportunity between the first white space character in the sequence and whatever comes before it, to avoid wrapping a single space to the next line (I think you were the first to suggest that that would be a bad idea).